config/bidderinfo.go

```diff
@@ -275,11 +276,29 @@ func (infos BidderInfos) validate(errs []error) []error {
 			if validateSyncerErr != nil {
 				errs = append(errs, validateSyncerErr)
 			}
+
+			validateAliasesErr := validateAliases(bidder, infos, bidderName)
+			if validateAliasesErr != nil {
+				errs = append(errs, validateAliasesErr)
+			}
```

Comment:
Nitpick: The error check can be added to the same line as the assignment for a slight code style optimization:

```
if e := validateAliases(bidder, infos, bidderName); e != nil {
	errs = append(errs, e)
}
```

Reply:
Fixed here - https://github.com/prebid/prebid-server/pull/2942/commits/1b10e0664831096078e59f1eec08b92e23a5fbd9